assets/gql/contacts/import.gql

```diff
@@ -1,7 +1,7 @@
 #import "./ImportFields.frag.gql"
 
-mutation importContacts($group_label : String, $data : String) {
-  importContacts(group_label: $group_label, data: $data) {
+mutation importContacts($group_label : String, $data : String, $type : UploadContactsTypeEnum, $id : ID!) {
```

Comment:
Can we keep the naming same?
Either upload or import

Reply:
@pankaj-ag made requested changes